dissect/target/plugins/apps/webserver/apache.py

```diff
@@ -77,83 +196,165 @@ def get_log_paths(self) -> list[Path]:
             - https://unix.stackexchange.com/a/269090
         """
 
-        log_paths = []
+        access_log_paths = set()
+        error_log_paths = set()
 
         # Check if any well known default Apache log locations exist
-        default_log_dirs = ["/var/log/apache2", "/var/log/apache", "/var/log/httpd", "/var/log"]
-        default_log_names = ["access.log", "access_log", "httpd-access.log"]
-        for log_dir, log_name in itertools.product(default_log_dirs, default_log_names):
-            log_paths.extend(self.target.fs.path(log_dir).glob(log_name + "*"))
-
-        # Check default Apache configs for their CustomLog directive
-        default_config_paths = [
-            "/etc/apache2/apache2.conf",
-            "/usr/local/etc/apache22/httpd.conf",
-            "/etc/httpd/conf/httpd.conf",
-        ]
-
-        for config in default_config_paths:
+        for log_dir, log_name in itertools.product(self.DEFAULT_LOG_DIRS, self.ACCESS_LOG_NAMES):
+            access_log_paths.update(self.target.fs.path(log_dir).glob(log_name + "*"))
```

Comment:
```suggestion
            access_log_paths.update(self.target.fs.path(log_dir).glob(f"{log_name}*"))
```
This reads a bit nicer IMO.